### ISSUE TITLE
Windows Server 2008 Standard sp2 not working

### ISSUE THREAD

**Issue by berlinsaint** [martinlindhe/wmi_exporter#122]
On my two 2008 Standar windows, wmi_exporter doesnt work . when listening the port, it doesn't deal any connections.

**Comment by carlpett** [comment]
Hi @berlinsaint,
Can you reach the endpoint on localhost, or is it unresponsive there as well?
Do you get any output when running the exporter?

**Comment by berlinsaint** [comment]
hi @carlpett  i can't reach the endpoint on localhost, and when i access the localhost:9182 , the console dosen't have any change, i check the netstat status, and it did listen the port 9182, very strange.
but on my another three 2008 professional Edition , they have no problem. i doubt that the exporter dosen't match the  2008 standard version.
-----------------------
another try is that i change the listen port to 5001,and after a longtime no output;the console output this:
![image](https://user-images.githubusercontent.com/10629930/30788583-e1c0df18-a1d0-11e7-86e6-07512f8b5967.png)

**Comment by carlpett** [comment]
@berlinsaint Could you translate the error messages given in the screenshot, please? 
But something is clearly going wrong here, and taking much too long time to do so. We should not let a query run for 1400+ seconds, or even 10+ seconds. So that will be a separate thing we should fix, but it is not the direct cause of your problems, I think.

**Comment by berlinsaint** [comment]
the above chinese words means : “occur an accident， server running with failure”

**Comment by berlinsaint** [comment]
can anyone help me ?

**Comment by carlpett** [comment]
Sorry, been traveling. Still, it is a very odd error, with very little to go on. Could you try running this in Powershell: `Get-WmiObject Win32_OperatingSystem`?

**Comment by berlinsaint** [comment]
yes i run the command , and wait a long long time with not any output. how to solve it ?
![image](https://user-images.githubusercontent.com/10629930/31529890-d5f54722-b00e-11e7-872d-a94d4772df84.png)

**Comment by carlpett** [comment]
Then you likely have some issue with the WMI system on your machine. You'll need to do some troubleshooting, you can take a look at [this issue](https://github.com/martinlindhe/wmi_exporter/issues/50#issuecomment-270630204) for a starting point.

**Comment by carlpett** [comment]
@berlinsaint Did you come to some conclusion about your problem?
